ssl/src/main/java9/org/wildfly/security/ssl/JDKSpecific.java

```diff
@@ -0,0 +1,34 @@
+package org.wildfly.security.ssl;
```

Comment:
Just a very minor comment, this is missing a copyright header.

Reply:
fixed. thank you